addon/-private/execution_context/rfc268.js

```diff
@@ -0,0 +1,163 @@
+import $ from 'jquery';
```

Comment:
we should use `-jquery` here cause it can be ran in consumers test suites w/o `Ember.$` available